demo/CMakeLists.txt

```diff
@@ -34,7 +37,10 @@ endif()
 
 # add dynamic demo
 add_executable(demo main.cc chainmain.cc cronos.cc extra.cc third_party/easywsclient/easywsclient.cpp)
-if (WIN32)
+if (WIN32 AND MSYS)
+  target_link_libraries(demo PUBLIC ${PLAY_CPP_SDK_LIB} ${RUST_LIB} Ncrypt userenv ntdll Secur32 crypt32 ws2_32 ntdll)
```

Comment:
RUST_LIB is unnecessary. It is already link into PLAY_CPP_SDK_LIB by https://github.com/cronos-labs/play-cpp-sdk/blob/main/demo/sdk/CMakeLists.txt#L22

PLAY_CPP_SDK_LIB is a output of SDK/CMakeLists.txt.

Reply:
We shouldn't link RUST_LIB in demo/CMakeLists.txt which is not portable. As a whole, RUST_LIB and play_cpp_sdk.lib, and play_cpp_sdk.dll.lib should be bundled together inside sdk/CMakeLists.txt.

Reply:
Ok, I got it, you are planing to link static library, but here is linking to dynamic library. You could create a new CMakeLists.txt to do that instead of overriding the dynamic library linking. 

See
https://github.com/cronos-labs/play-cpp-sdk/blob/main/demo/demostatic/demostatic.vcxproj#L171
https://github.com/cronos-labs/play-cpp-sdk/blob/main/demo/demo/demo.vcxproj#L174-L175

Reply:
yes, it should work.
i'll check